Translate homepage (#17)
reactjs/nl.reactjs.org: content/home/examples/a-simple-component.md

---

> title: Een Eenvoudige Component

Reviewer: Eenvoudig?

Reviewer 2: @duivvv If you're commenting on the need for the additional "e", I think it's necessary here because of the Dutch word "component"'s genus.

Reviewer: Looks very weird, but I trust you on this👍

order: 0
domid: hello-example
---

> React componenten implementeren een `render()` functie die input data inneemt en terug geeft wat er weergegeven moet worden.

Reviewer 3: ```suggestion
React componenten implementeren een `render()` functie die input data accepteert en terug geeft wat er weergegeven moet worden.
```

Reviewer 2: I would suggest: 

```suggestion
React componenten implementeren een `render()` functie die input data aanneemt en teruggeeft wat er weergegeven moet worden.
```

> Dit voorbeeld gebruikt de XML-achtige syntax genaamd JSX. 

Reviewer 3: ```suggestion
Dit voorbeeld gebruikt een XML-achtige syntax genaamd JSX. 
```

Reviewer 2: I agree with @ronderksen's suggestion here.

Author: 👍

> Input data die wordt doorgegeven aan het component is voor `render()` toegankelijk via `this.props`.

Reviewer 3: ```suggestion
Input data die wordt doorgegeven aan het component is in de `render()` functie toegankelijk via `this.props`.
```

Reviewer 2: I think both versions are fine.

Author: Mine is probably more suited towards experienced developers that know what it means, but @ronderksen's version is more clear for beginners as well. I will take the suggestion
